x-pack/plugins/translations/translations/ja-JP.json

```diff
@@ -7420,7 +7420,6 @@
     "xpack.ml.calendarsEdit.newEventModal.startDateAriaLabel": "開始日",
     "xpack.ml.calendarsEdit.newEventModal.toLabel": "終了:",
     "xpack.ml.calendarsList.deleteCalendars.calendarsLabel": "{calendarsToDeleteCount} 件のカレンダー",
-    "xpack.ml.calendarsList.deleteCalendars.deletingCalendarErrorMessage": "カレンダー {calendarId} の削除中にエラーが発生しました: {errorMessage}",
```

Comment:
Isn't this translation still being used?

Reply:
yes, but the text has changed. so i think it's marked it as removed so the translated text can be remade.